pkg/model/catalog.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package model
```

Comment:
Naive question: does CF define a swagger spec for this API? If so, does it make sense to auto-generate these definitions and corresponding go bindings with https://github.com/go-swagger/go-swagger? An added benefit to this approach would be that the test client (brkc) could be implemented using the generated client code from the same swagger spec.